docs/wallets.md

```diff
@@ -28,18 +28,18 @@ codex32 shares may be any length between 128 and 512 bits.
 Wallets should support import of 128- or 256-bit seeds; other lengths are optional.
 
 128-bit seeds are 48 characters in length, including the `MS1` prefix.
-256-bit seeds are 74. For other bitlengths, see the BIP.
+256-bit seeds are 74. For other bit-lengths, see the BIP.
 
 The process for entering shares is:
 
-1. The user should select the bit length of the import before entering any actual data.
-1. Then the user should enter the first share. To the entext possible given screen limitations, when entering share data, data should be displayed in uppercase, visually separated into four-character windows. The first four-character window should include the `MS1` prefix, which should be pre-filled.
+1. The user should enter the first share. To the extent possible given screen limitations, when entering share data, data should be displayed in uppercase, with visually distinct four-character windows. The first four-character window should include the `MS1` prefix, which should be pre-filled.
```

Comment:
For subsequent shares the threshold and identifier (first 8 characters will also be pre-filled).

Reply:
I already have "the wallet may pre-fill...". Should I change this from "may" to "must"?

Reply:
optionally the pre-filled 'ms1' can be disabled for editing as well.